govcd/tar.go

```diff
@@ -0,0 +1,106 @@
+package govcd
```

Comment:
We should create a separate package for functionalities that are not part of the library (file handling, compression/decompression utilities, logging, etc.)
I suggest calling it "util"

Reply:
As we got confirmation in email -> made the change.